Quizzler-iOS13/Controller/ViewController.swift

```diff
@@ -8,7 +8,7 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+public class ViewController: UIViewController {
```

Comment:
Why do you need public? You going to use ViewController inside the same module, so should be internal (default)

Reply:
Alright. I thought the default class was public. 

I still have the same issue on "Failed to instantiate the default view controller for UIMainStoryboardFile 'Main' - perhaps the designated entry point is not set?" and the app stays on a black screen.

Reply:
![image](https://github.com/ronaldngounou/Quizzler-iOS13/assets/74538524/7308bf6d-037b-4f7c-8997-2180dae85282)

I saw your comment on removing references to Main.storyboard in the info.plist. I deleted the two lines underlined in red. 
I also tried to change the "LaunchScreen interface base name to" ViewController but the screen remains black. 
Thank you!

Reply:
There are 2 ways to fix it:

1. Restore SceneDelegate and then initialize window like this
`func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {

    if let windowScene = scene as? UIWindowScene {
        let window = UIWindow(windowScene: windowScene)
        // window.rootViewController = ...
        self.window = window
        window.makeKeyAndVisible()
    }
}`

2. Get rid of SceneDelegate approach, for that you need to also remove Application Scene Manifest from Info.plist

1st approach is preferable though

> I also tried to change the "LaunchScreen interface base name" to ViewController but the screen remains black.

Keep it as it was :)

Reply:
I followed the first approach. I had to copy and paste the old SceneDelegate file from a previous state and create a new file. I searched how to restrore a project from a state but I was not able to figure out how to show the terminal in xcode. 

In my latest commit [94b81cf](https://github.com/ronaldngounou/Quizzler-iOS13/pull/2/commits/94b81cfd52ab2e91bbfce49128083cffdbb53a0b), I have some issues regarding nil values.